### PR TITLE
contact_rollups schema changes

### PR DESCRIPTION
**Pull request by jeremydstone**
This PR makes two changes to the contact_rollups schema:
- Adds an index to `pardot_id`, which it turns out we need to look up by
- Adds a new `email_malformed` column, which we use to track email addresses that are rejected by Pardot so we don't fruitlessly try to re-sync the email address forever

Submitting the schema changes by themselves in this PR per best practice. Code that takes advantage of the new column coming in a subsequent PR.

### CODE REVIEW

File: pegasus/migrations/116_add_contact_rollups_email_malformed_column.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+Sequel.migration do
+  change do
+    alter_table(:contact_rollups) do
+      add_column :email_malformed, TrueClass
```
Comment (aoby) [inline pegasus/migrations/116_add_contact_rollups_email_malformed_column.rb:4]: I had to look up what TrueClass does in this [context](https://github.com/jeremyevans/sequel/blob/master/doc/schema_modification.rdoc). 
> for boolean columns, you can use either TrueClass or FalseClass, they are treated the same way (ruby doesn't have a Boolean class)

**Comment by aoby** [comment]
Any reason these are 2 separate migrations? I'm pretty sure they can be combined into a single file.

File: pegasus/migrations/115_add_contact_rollups_pardot_id_index.rb [hunk 2]
```diff
@@ -0,0 +1,7 @@
+Sequel.migration do
+  change do
+    alter_table(:contact_rollups) do
+      add_index :pardot_id
```
Comment (ashercodeorg) [inline pegasus/migrations/115_add_contact_rollups_pardot_id_index.rb]: Should this index be unique? Presumably, we don't want to rows describing the same Pardot contact?

**Comment by ashercodeorg** [comment]
LGTM. Fine with separate migrations or one migration, I think there are arguments to be made for both.

File: pegasus/migrations/115_add_contact_rollups_pardot_id_index.rb (hunk 2 above)
Comment (jeremydstone) [reply pegasus/migrations/115_add_contact_rollups_pardot_id_index.rb]: Good idea, done

**Comment by jeremydstone** [comment]
I saw past examples of both combining and not combining things like this... given we tend to emphasize the spirit of "separate things out" I tipped that way.

**Pull request merged**